### ISSUE TITLE
Can't disable the word wrapping for a header of a column

### ISSUE THREAD

**Issue by spxcal** [schoetbi/XPTable#50]
Hi Developers,

This is more of a question rather than an issue, I noticed this when I upgraded the XPTable from 1.02.0 to 1.6.1.139.

The column headers have word wrapping by default after the upgrade. Given a fixed width, the content will show up like below. I would expect the text is trimmed and appears as `...` when there is not enough space. 
![image](https://github.com/schoetbi/XPTable/assets/136601173/79622624-5e7c-4612-ade4-8b67acc57a7d)

I tried to disable the word wrapping in the table level but that seems not applicable for the column header. Also tried to enable the tooltip in the table level and set the IsTextTrimmed in the column level to allow trim text as per HeaderRenderer logic, which didn't work for me either. The cells themselves are working fine but I didn't find such property or method that allows me to modify the column headers

I was wondering if I missed something or there is any work around to fix this other than increasing the width? 

Many thanks!
spxcal

**Comment by schoetbi** [comment]
Hi spxcal, since we are no longer developing new things with WinForms, the development of XPTable has been somewhat discontinued. So I am no longer "in the code" and unfortunately do not have the time to go through it and test it. Sorry about that!

**Comment by spxcal** [comment]
Hi Schoetbi, that's fine, thank you for your reply! I will try something else. Cheers ;)

**Comment by adarmus** [comment]
Hi - sorry - have been on holiday. I have added a branch that implements a WordWrap property on the XPHeaderRenderer that (I think) gives you the control you want. 

I haven't made a PR as to get things working I had to mess around with the project files, and I don't have the time to get to the bottom of what is needed. This may give you enough to get it working.

https://github.com/schoetbi/XPTable/tree/columntextwrap
